### PR TITLE
Typography audit

### PR DESCRIPTION
**Pull request by joodaloop**
there are three pages (Tags, Archive, Default) with inline styles, because those are one-off styles that don't make sense to encapsulate in a class

### CODE REVIEW

File: .jekyll-metadata
Comment (pepijndevos) [inline .jekyll-metadata:1]: Should not be commited

https://jekyllrb.com/docs/structure/

> It’s probably a good idea to add this to your .gitignore file.

File: _layouts/default.html [hunk 1]
```diff
@@ -21,47 +37,55 @@
...
+
+                    <h4  style="margin-top: 1.5em;">Elsewhere on the internet...</h4>
+                    <ul>
+                            <!--<li><a href="http://eepurl.com/iKGOdY">Mailinglist</a></li>-->
+                            <li><a href="http://twitter.com/pepijndevos" target="_blank">Twitter</a></li>
+                            <li><a href="http://github.com/pepijndevos" target="_blank">Github</a></li>
+                    </ul>
+
+                    {% if site.related_posts %}
+                    <h4 style="margin-top: 1.5em;">Remotely related posts</h4>
```
Comment (pepijndevos) [inline _layouts/default.html:70]: Really don't like these, especially the ones that are clearly not one-of but repeat several times.

File: style/style.css [hunk 2]
```diff
@@ -6,72 +6,225 @@ Version: 1.0
...
+:root{
+    --background: #FFF;
+    --text: #151515;
+    --grey: lightgray;
+    --theme: #227;
+    --theme-light: rgba(240, 240, 247, 1); /* #22227711; */
+    --main-width: 620px;
+    --padding: 3em;
+    --serif: Newsreader;
+    --sans: Hind;
```
Comment (pepijndevos) [inline style/style.css:18]: So what do I put here to reproduce the different options you were playing with?

File: _layouts/default.html (hunk 1 above)
Comment (stealsocks) [reply _layouts/default.html:70]: they are repeated exactly two times lmao

File: style/style.css (hunk 2 above)
Comment (stealsocks) [reply style/style.css:18]: the --serif will change the body text, the --sans will change the sidebar and data/tag fonts

**Pull request merged**